### ISSUE TITLE
WCAG for content that is only shown?

### ISSUE THREAD

**Issue by kengdoj** [w3c/remote-meetings#17]
Apologies, I opened [this issue](https://github.com/w3c/apa/issues/245) in the wrong repo:

Question about this part -

> [4.1.1 Relevance of the Web Content Accessibility Guidelines](https://raw.githack.com/w3c/apa/accessible-meetings/remote-meetings/Overview.html#w3c-guidance-relevant-for-accessible-remote-meetings)
> Any prepared content (e.g., documents, presentation slides, prerecorded multimedia) that is shared with or shown to meeting participants is subject to the Web Content Accessibility Guidelines (WCAG). Policies typically specify that documents, presentations and related materials should conform to WCAG 2.1 Level AA.

If content is only shown (not shared) in a remote meeting, participants are not interacting with the content directly so it seems that only some WCAG SCs should be required - like contrast. SC's that require name, role, value (for example) wouldn't be useful if the file is not shared.

**Comment by jasonjgw** [comment]
I created pull request #18 to address this issue.

It is written in such a way that solutions which present the materials using Web-based technologies (Microsoft's, for instance) are covered.

Comments on the changes are welcome.

**Comment by sehollier** [comment]
Closing Issue as now moved to Issue 18. Support the change.
